nh_eobs_backup/models/base_config_settings.py

```diff
@@ -0,0 +1,65 @@
+from openerp.osv import osv, fields
+
+
+class NHClinicalObservationBackupSettings(osv.TransientModel):
+    """
+    Add settings for the list of wards to backup
+    """
+
+    _inherit = 'base.config.settings'
+    _name = 'base.config.settings'
+
+    _columns = {
+        'locations_to_print': fields.many2many(
+            'nh.clinical.location',
+            domain=[['usage', '=', 'ward']],
+            string='Locations to print backup observation reports for'
+        )
+    }
+
+    def set_locations(self, cr, uid, id, context=None):
+        """
+        Set the locations to backup. This is a 'magical' method name used by
+        Odoo for setting the value of fields exposed on the GUI
+
+        :param cr: Odoo Cursor
+        :param uid: User ID
+        :param ids: IDs of the setting wizard
+        :param context: Odoo context
+        :return: Result of writing field
+        """
+        loc_pool = self.pool.get('nh.clinical.location')
```

Comment:
Should we change this to just be normal key access? If for some reason `nh.clinical.location` doesn't exist then let's just fail immediately because something's wrong.